robot/Cumulus/tests/browser/engagement_plans/edit_engagement_plan.robot

```diff
@@ -2,34 +2,54 @@
 
 Resource        robot/Cumulus/resources/NPSP.robot
 Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/ContactPageObject.py
+...             robot/Cumulus/resources/AccountPageObject.py
 ...             robot/Cumulus/resources/EngagementPlanTemplatesPageObject.py
-Suite Setup     Open Test Browser
+...             robot/Cumulus/resources/NPSP.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
 Suite Teardown  Delete Records and Close Browser
 
+***Keywords***
+# Setup a contact with parameters specified
+Setup Test Data
+    Setupdata   engagement  engagement_data=${engagement_fields}
+
+
 *** Variables ***
-${task3_1}  Follow-Up Phone Call3
+${task1}  Task_1
+${task2}  Task_2
+&{engagement_fields}  Name=AutomationTestPlan
 
 
 *** Test Cases ***
+Create Engagement Plan And Edit the Plan
+    [Documentation]                      Create an engagement plan , Edit the Engagement plan and add tasks
+    ...                                  Verify the engagement plan added appears in the list with the edited
+    ...                                  tasks.
+    [tags]                               W-038641                 feature:Engagements
+
+
+    Go To Page                                        Listing                             Engagement_Plan_Template__c
+    Go To Engagement Plan Page                        Edit                                ${data}[engagement][Id]
+    Wait For Locator                                 frame                               Manage Engagement Plan Template
+
+    Select Frame And Click Element                   Manage Engagement Plan Template     id    idName
+
+    Click Button                                     Add Task
+    Wait Until Page Contains                         Task 1
+    Enter Task Id and Subject                        Task 1                             Task_1
+    Click Button                                     Add Task
+    Wait Until Page Contains                         Task 2
+    Enter Task Id and Subject                        Task 2                             Task_2
+    Page Scroll To Locator                           button                             Save
+    Click Button                                     Save
```

Comment:
same comments as above on waits, frame and scroll

Reply:
Done